package.json

```diff
@@ -0,0 +1,83 @@
+{
+    "name": "@bancor/dad-bridge-wrapper",
+    "version": "1.0.0",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/bancorprotocol/dad-bridge-wrapper.git"
+    },
+    "homepage": "https://github.com/bancorprotocol/dad-bridge-wrapper#readme",
+    "bugs": {
+        "url": "https://github.com/bancorprotocol/dad-bridge-wrapper/issues"
+    },
+    "author": "Barak Manos",
+    "license": "SEE LICENSE IN LICENSE",
+    "files": [
+        "/contracts/**/*.sol",
+        "/artifacts/contracts/**/*.json",
+        "/typechain/**/*.ts"
+    ],
+    "scripts": {
+        "hh": "hardhat",
+        "build": "hardhat compile",
+        "rebuild": "yarn clean && yarn build",
+        "test": "NODE_OPTIONS=${CI:+'--max-old-space-size=8192'} hardhat test",
+        "testb": "BAIL=1 yarn test",
+        "size": "yarn rebuild && hardhat size-contracts",
+        "verify": "hardhat verify",
+        "coverage": "yarn build && NODE_OPTIONS=${CI:+'--max-old-space-size=8192'} hardhat coverage",
+        "flatten": "hardhat flatten",
+        "lint": "solhint --max-warnings 0 contracts/**/*.sol",
+        "format": "prettier --check --write contracts/**/*.sol test/**/*.ts --config .prettierrc",
+        "snyk-protect": "snyk protect",
+        "prepare": "yarn build && yarn snyk-protect",
+        "clean": "rm -rf artifacts cache data coverage typechain"
+    },
+    "dependencies": {
+        "hardhat": "2.6.5"
+    },
+    "devDependencies": {
+        "@ethersproject/hardware-wallets": "^5.4.0",
```

Comment:
pretty sure we don't need this huge list

Reply:
Yeah I've removed the ones that I know for sure aren't needed.

Reply:
ok

Reply:
Yeah, many are actually unneeded. Let's clean this up after everything else is finished